include/nbl/asset/utils/IVirtualTexture.h

```diff
@@ -139,7 +139,7 @@ class IVirtualTexture : public core::IReferenceCounted, public IVirtualTextureBa
         uint64_t pgTab_x : 8;
         uint64_t pgTab_y : 8;
         uint64_t pgTab_layer : 8;
-        uint64_t maxMip : 4;
+        uint64_t maxMip : 4; // value 0x0fu means, the texture takes only 1 page of physical storage (miptail)
```

Comment:
Why not have it encode the old value +1

Then miptail only would be 0, and everything else would be sane (less special cases)

Reply:
i dont see how it would change anything, then special value is 0 instead of -1 and thats it

Reply:
see the changes to the shader which are possible